### DISCUSSION TITLE
How do you merge an animated mesh with Crest Ocean water so that they seamlessly visually combine?

### DISCUSSION THREAD

**Discussion by hmatt843** [crest-ocean/crest#1137] [Q&A]
I have an animated mesh (a skeleton dancing, haha) that I want to make part of Crest Ocean water. I have Crest Ocean water already in the scene, and I'd like to be able to turn the animated skeleton mesh into Crest water so that the skeleton blends in seamlessly visually with the Crest ocean water. 

_Additionally_, ideally the buoyancy would still work so that the buoyant objects could float up from the regular Crest water into the skeleton mesh water. But this is probably harder to accomplish than the above task.

How does one achieve this? 
Please help/advise!

**Comment by magenta404** [comment]
Inputs render from a top-down perspective so the best you can do is add the skeleton as height to the water which is probably not what you're after. You can try it by adding a RegisterHeightInput to the GO with the renderer on it. I haven't tested a SkinnedMeshRenderer, if that is what you're using, with this system so not sure if it will work.
